client/src/components/advancedSearch/DateRangeFilter.tsx

```diff
@@ -116,6 +124,9 @@ const DateRangeFilter = ({
       moment(value.end).format(Settings.dateFormats.forms.displayShort.date)
     )
   }
+  if (value.relative === LAST_X_DAYS) {
+    dateRangeDisplay = `Last ${value.days || "X"} days`
```

Comment:
Suggest to change to `Last ${value.days ?? "?"} days` (`0` might be valid in some [future] cases), and `X` can be confusing (Roman value for 10).